pytorch_lightning/utilities/distributed.py

```diff
@@ -345,3 +346,26 @@ def register_ddp_comm_hook(
 
 def tpu_distributed() -> bool:
     return _TPU_AVAILABLE and xm.xrt_world_size() > 1
+
+
+def init_ddp_connection(
+    cluster_environment: ClusterEnvironment,
+    torch_distributed_backend: str,
+    global_rank: Optional[int] = None,
+    world_size: Optional[int] = None,
+) -> None:
+    global_rank = global_rank if global_rank is not None else cluster_environment.global_rank()
+    world_size = world_size if world_size is not None else cluster_environment.world_size()
+    os.environ["MASTER_ADDR"] = cluster_environment.master_address()
+    os.environ["MASTER_PORT"] = str(cluster_environment.master_port())
+    if torch.distributed.is_available() and not torch.distributed.is_initialized():
+        log.info(f"initializing ddp: GLOBAL_RANK: {global_rank}, MEMBER: {global_rank + 1}/{world_size}")
+        torch.distributed.init_process_group(torch_distributed_backend, rank=global_rank, world_size=world_size)
+
```

Comment:
init_process_group can take additional args, which we don't currently expose. should we make these settable with **kwargs? https://pytorch.org/docs/stable/_modules/torch/distributed/distributed_c10d.html#init_process_group

Reply:
We could pass the kwargs from the `Plugins` constructor and then, to `init_process_group` by checking its signature and pass the matching params.

Reply:
Would be a follow-up PR to support it.